contracts/Strategy.sol

```diff
@@ -2,15 +2,50 @@
 pragma solidity 0.8.14;
 
 import "@openzeppelin/contracts/utils/math/Math.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+import {IERC20, BaseStrategy, SafeERC20} from "BaseStrategy.sol";
 
-import {IERC20, BaseStrategy} from "BaseStrategy.sol";
 import "interfaces/IVault.sol";
 import {Comet, CometStructs, CometRewards} from "interfaces/CompoundV3.sol";
+import {ITradeFactory} from "interfaces/ySwaps/ITradeFactory.sol";
+import {ISwapRouter} from "interfaces/univ3/ISwapRouter.sol";
+
+contract Strategy is BaseStrategy, Ownable {
+    using SafeERC20 for IERC20;
 
-contract Strategy is BaseStrategy {
-    // TODO: add rewards
     Comet public immutable cToken;
-    uint256 private SECONDS_PER_YEAR = 365 days;
+    uint256 internal constant SECONDS_PER_YEAR = 365 days;
+    uint256 internal constant DAYS_PER_YEAR = 365;
+    uint256 internal constant SECONDS_PER_DAY = 60 * 60 * 24;
+
+
+    //Uniswap v3 router
+    ISwapRouter internal constant router =
+        ISwapRouter(0xE592427A0AEce92De3Edee1F18E0157C05861564);
+
+    uint24 public compToEthFee;
+    uint24 public ethToWantFee;
+
+    address internal constant comp = 
+        0xc00e94Cb662C3520282E6f5717214004A7f26888;
+
+    address internal constant weth = 
+        0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+
+    address public tradeFactory;
+
+    CometRewards public constant rewardsContract = 
+        CometRewards(0x1B0e765F6224C21223AeA2af16c1C46E38885a40);
+
+    address public constant COMP =
```

Comment:
Comp already is a global variable [here](https://github.com/jmonteer/yearnv3-strat-compoundv3-lender/pull/4/files#diff-3e804d467472e49fe8afea1899d96f26655f3ecd641555a801abcea4f12af598R30)